### ISSUE TITLE
Improve installer logging mechanism

### ISSUE THREAD

**Issue by Demonsthere** [kyma-project/kyma#8149]
**Description**

- Expand the current logging mechanism to support json format. 
- Add a yaml<->json parser to the code
- Add a flag for controlling output type: yaml | json
- Split the override logging into several messages instead of one big chunk 

**Reasons**

The kyma-operator (installer) prints logs using a yaml format, which allows easy helm integration. However, this format is nonstandard and difficult to handle from a log collection system point of view. Moreover, all overides are logged as a single yaml structure in a single log entry. This makes it very hard to parse (for human and machine alike)

**Attachments**

Current logs:

```
2020/04/22 11:09:03 Starting operator...
2020/04/22 11:09:03 Processing Kyma components
2020/04/22 11:09:03 ---------------------------
2020/04/22 11:09:03 install component cluster-essentials
2020/04/22 11:09:03 ---------------------------
2020/04/22 11:09:03 Get Kyma Sources
2020/04/22 11:09:03 Kyma sources available locally.
2020/04/22 11:09:03 Get Kyma Sources...DONE
2020/04/22 11:09:03 Overrides used for installation of component cluster-essentials
2020/04/22 11:09:03
 global:
  adminPassword: foobar
  applicationConnectorCa: ""
  applicationConnectorCaKey: ""
  domainName: kyma.local
  helm:
    ca:
      crt: |
```

**Acceptance criteria:**
- Overrides are logged in file or files (single file for all overrides, or per component)
- Logs are human-readable and serve debugging purposes
 - Potential ideas:
    - Use a different logger, that can serve txt, json, yaml
    - Prase override logs before printing them
    - Split logs into several messages instead of one chunk

**Comment by piotrmsc** [comment]
@Demonsthere still valid?

**Comment by strekm** [comment]
closing this one as installer will be replaced with https://github.com/kyma-project/kyma/issues/10297.
